src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -49,10 +49,12 @@ public function process(ContainerBuilder $container)
             $coreThemePath = dirname(dirname($reflClass->getFileName())).'/Resources/views/Form';
             $container->getDefinition('twig.loader.native_filesystem')->addMethodCall('addPath', array($coreThemePath));
 
-            $paths = $container->getDefinition('twig.cache_warmer')->getArgument(2);
-            $paths[$coreThemePath] = null;
-            $container->getDefinition('twig.cache_warmer')->replaceArgument(2, $paths);
-            $container->getDefinition('twig.template_iterator')->replaceArgument(2, $paths);
+            if ($container->hasDefinition('twig.cache_warmer')) {
+                $paths = $container->getDefinition('twig.cache_warmer')->getArgument(2);
+                $paths[$coreThemePath] = null;
+                $container->getDefinition('twig.cache_warmer')->replaceArgument(2, $paths);
+                $container->getDefinition('twig.template_iterator')->replaceArgument(2, $paths);
```

Comment:
either this service should be also removed when cache is disabled, or its configuration here should not be disabled when disabling the cache warming.